features/step_definitions/visitor_can_create_new_task_steps.rb

```diff
@@ -0,0 +1,24 @@
+Then("I should be on New Task page") do
+    visit new_task_path
+end
+
+Then("I fill in {string} with {string}") do |field, content|
+    fill_in field, with: content
+end
+
+Then("I click on {string} link") do |new_task|
```

Comment:
This variable name should be more generic. `new_task` is probably not the text you will have when you use this step def in another scenario.